### DISCUSSION TITLE
Mac OS problem

### DISCUSSION THREAD

**Discussion by m0roj** [AlexandreRouma/SDRPlusPlus#416] [Q&A]
Is this still WiP?
$ sdrpp
dyld: Library not loaded: /usr/local/opt/glew/lib/libGLEW.2.2.dylib
  Referenced from: /usr/local/bin/sdrpp
  Reason: image not found
Abort trap: 6
$

**Comment by AlexandreRouma** [comment]
you need to install the depdencies with homebrew, they're the same as linux
